src/analyzer/git-caller.server.ts

```diff
@@ -264,6 +277,7 @@ export class GitCaller {
       `--max-count=${count}`,
       this.branch,
       "--numstat",
+      // "--cc", // include file changes for merge commits
```

Comment:
Please remove this commented-out code.

Reply:
I would like to keep it in case we change our mind about including merge commits, or want to make it an option or something like that